apps/concept-catalog/pages/[[...catalogId]]/index.tsx

```diff
@@ -7,6 +7,7 @@ import {
   PageSubtitle,
   Button,
   Icon,
+  SearchHit,
 } from '@catalog-frontend/ui';
 import {localization} from '@catalog-frontend/utils';
 import SC from '../../styles/search-page';
```

Comment:
Bruk css modules frem for styled components

Reply:
Så det er ikke noe problem å blande i samme fil? Eller mener du å skrive om det som finnes der til før så alt er css modules?